usaspending_api/common/matview_manager.py

```diff
@@ -12,6 +12,7 @@
 OVERLAY_VIEWS = [
     settings.APP_DIR / "database_scripts" / "matviews" / "vw_award_search.sql",
     settings.APP_DIR / "database_scripts" / "etl" / "transaction_delta_view.sql",
+    settings.APP_DIR / "database_scripts" / "etl" / "award_delta_view.sql",
```

Comment:
In Seth's PR he removed the pre-existing ES view. I don't think you'll need this new one listed here

Reply:
It's necessary for running the tests, the award index tests error out without them, but I'll look and see if there's a different way to get the view to create

Reply:
It was necessary, it shouldn't be necessary after Seth made changes in the other PR. It might require some changes to the test config functions